metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/DateFunctionsTest.java

```diff
@@ -254,7 +254,7 @@ public void testDateFormatNow() {
     Object result = run("DATE_FORMAT('EEE MMM dd yyyy hh:mm:ss zzz', 'GMT')");
     assertTrue(result.toString().endsWith("GMT"));
   }
-
+/*
```

Comment:
Looks like you may have inadvertently left in the commented out bits here. I'm assuming this is related to the timezone related bug, which I committed yesterday - https://github.com/apache/metron/pull/1487. Undo this, merge in master, and the tests should be ready to go.